### PR TITLE
Inventory system

### PR DESCRIPTION
**Pull request by fourlastor**
https://user-images.githubusercontent.com/1263058/201484446-f6320529-56d3-4a53-9670-ce4238d70b8e.mp4

### CODE REVIEW

**Comment by fourlastor** [comment]
trigger format

File: player/Player.tscn [hunk 1]
```diff
@@ -223,8 +223,6 @@ start_node = "Idle"
 
 [node name="Player" type="KinematicBody2D"]
 script = ExtResource( 2 )
-speed = 100.0
-runningSpeed = 130.0
```
Comment (fourlastor) [inline player/Player.tscn:227]: I think these got removed as it's the default value

File: player/Player.tscn [hunk 2]
```diff
@@ -5,7 +5,7 @@
 [ext_resource path="res://player/player-Sheet.png" type="Texture" id=3]
 
 [sub_resource type="CapsuleShape2D" id=42]
-radius = 8.0
+radius = 6.0
```
Comment (fourlastor) [inline player/Player.tscn:8]: Reduced the radious as the player would get stuck in 1 tile wide corridors

File: project.godot [hunk 3]
```diff
@@ -31,6 +31,12 @@ snap=false
 
 common/drop_mouse_on_gui_input_disabled=true
 
+[importer_defaults]
+
+texture={
+"flags/filter": false
+}
+
```
Comment (fourlastor) [inline project.godot:39]: This avoids us to change the import of every image to be scaled with nearest neighbour

**Review by takecare — APPROVED** [review]

**Pull request merged**